read.go

```diff
@@ -323,7 +323,17 @@ func readNativeFrames(d dicomio.Reader, parsedData *Dataset, fc chan<- *frame.Fr
 	}
 
 	bytesRead = bytesAllocated * samplesPerPixel * pixelsPerFrame * nFrames
-
+	if uint32(bytesRead) == vl-1 {
```

Comment:
nit: This may be a little overkill b/c I don't know if it will happen (it probably shouldn't), but if vl=0 then `vl-1` will evaluate to `4294967295` because it's a uint32 which isn't really what we intend (even though it may not happen).

While it may not matter, from a correctness and safety perspective it may be better to write:
```suggestion
	if vl > 0 && uint32(bytesRead) == vl-1 {
```